### ISSUE TITLE
Bug with H2

### ISSUE THREAD

**Issue by stebrech** [billryan/gitbook-plugin-tags#4]
This plugin is exactly what I looked for. Thanks a lot for developing this!

I found this bug: My article start with a `h2` because the title of the article is already a `h1`.  The plugin changes my `h2` to a `h1` and puts a `#` in front of it.

![document_ist_nicht_tag_auf_oberster_ebene_ _barrierefreies_pdf](https://user-images.githubusercontent.com/2923268/28151615-d70d10d8-679b-11e7-95bc-9cf088538566.jpg)

**Comment by billryan** [comment]
Could you paste your raw markdown text here?

**Comment by stebrech** [comment]
Sure, there it is:

```
---
tags: Microsoft Word
---

## Problembeschreibung

Ein aus Word 2013 konvertiertes PDF enthält als oberstes Container das `<Part>` Tag. Die oberste Ebene sollte jedoch mit einem `<Document>` Tag strukturiert werden.

Problem wurde festgestellt bei der PDF-Konvertierung aus:

* Microsoft Word 2013 für Windows

### PAC 2 Warnung

> Part structure element used as root element

### Prüfpunkt/e Matterhorn Protokoll

> **01-006** Der Strukturtyp und Attribute eines Strukturelementes sind nicht semantisch geeignet für das Strukturelement. Alle Strukturelemente müssen in Betracht gezogen werden.  
> **→ interaktive/semantische Prüfung**

---

## ![](/assets/icon_acrobat.gif) Manueller Lösungsweg 1 in Acrobat

1. Das betroffene Tag im Tagbaum einmal anklicken um es auszuwählen.
2. Mit der Maus erneut auf das ausgewählte Tag klicken um es umzubenennen.
3. Den Tag `<Part>` in `<Document>` umbenennen.

![Animation zeigt die drei Schritte des manuellen Lösungswegs 1 in Acrobat](/assets/pdf-tag-umbenennen.gif)

## ![](/assets/icon_acrobat.gif) Manueller Lösungsweg 2 in Acrobat

1. Rechtsklick mit der Maus auf das betroffene Tag im Tagbaum.
2. Im Menü die Eigenschaften auswählen.
3. Im Eigenschaften-Fenster, im Register Tag, kann der Typ auf _«Dokument»_ \(übersetzte Schreibweisung\) geändert werden.

![Animation zeigt die drei Schritte des manuellen Lösungswegs 2 in Acrobat](/assets/pdf-tag-umbennnen-2.gif)
```
By the way, I'm using `theme-faq`. I do not have any other plugins installed.

**Comment by billryan** [comment]
Thanks for reporting this bug, I will fix it this weekend.

**Comment by stebrech** [comment]
Awesome, thanks so much!

**Comment by billryan** [comment]
Fixed in latest version 1.5.1. But the css looks uncomfortable for theme-faq.

**Comment by stebrech** [comment]
For me that is totally fine. I have some additional css anyway. Thanks again!
